src/common/localization/en-us.ts

```diff
@@ -202,6 +202,14 @@ export const english: IAppStrings = {
         downloadScript: "Analyze with python script",
         defaultLocalFileInput: "Browse for a file...",
         defaultURLInput: "Paste or type URL...",
+        editAndUploadToTrainingSet: "Edit & upload to training set",
+        editAndUploadToTrainingSetNotify: "by clicking on this button, this form will be added to the Azure Storage Blob for this project, where you can edit these labels.",
```

Comment:
We support local file system as a connection, not just Azure storage. So this message should reflect that

Reply:
updated to: "by clicking on this button, this form will be added to this project, where you can edit these labels."